<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialogClass</class>
 <widget class="QDialog" name="LoginDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open Profile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblProfile">
       <property name="text">
        <string>Profile:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbProfile"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblPassword">
       <property name="text">
        <string>Password:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lnePassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbtProfiles">
       <property name="text">
        <string>Settings...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dbbButtons">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cmbProfile</tabstop>
  <tabstop>lnePassword</tabstop>
  <tabstop>pbtProfiles</tabstop>
  <tabstop>dbbButtons</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
